### PR TITLE
Stop this fix from touching paid content

### PR DESCRIPTION
**Pull request by scottp719**
## What does this change?

Stops my other fix from having any impact on the paid content styled media pages
## Does this affect other platforms - Amp, Apps, etc?

Nopers, checked amp tests and all good.

## Request for comment

@regiskuckaertz

### CODE REVIEW

File: static/src/stylesheets/module/content/tones/_tone-media.scss [hunk 1]
```diff
@@ -58,6 +58,7 @@
         }
     }
 
+    &:not(.content--advertisement-feature)
     .brandbadge__logo {
```
Comment (gtrufitt) [inline static/src/stylesheets/module/content/tones/_tone-media.scss]: Can put this on the same line as it's part of the same selector:

`&:not(.content--advertisement-feature) .brandbadge__logo {`

Comment (ScottPainterGNM) [inline static/src/stylesheets/module/content/tones/_tone-media.scss]: Sweet tips, have made the change

**Review by regiskuckaertz — APPROVED** [review]
Agree with @gtrufitt but otherwise let's roll this out!

**Comment by ScottPainterGNM** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;LvH2gjdn&#x2F;145-fix-for-badges-on-paid-for-media-pages-fixing-my-fix"> Fix for badges on paid for media pages (fixing my fix)</a>

**Pull request merged**